catalogs/catalog-lakehouse-paimon/src/main/java/org/apache/gravitino/catalog/lakehouse/paimon/PaimonTablePropertiesMetadata.java

```diff
@@ -35,15 +35,26 @@
 public class PaimonTablePropertiesMetadata extends BasePropertiesMetadata {
 
   public static final String COMMENT = "comment";
-  public static final String CREATOR = "creator";
+  public static final String OWNER = "owner";
+  public static final String BUCKET_KEY = "bucket-key";
+  public static final String MERGE_ENGINE = "merge-engine";
+  public static final String SEQUENCE_FIELD = "sequence.field";
+  public static final String ROWKIND_FIELD = "rowkind.field";
+  public static final String PRIMARY_KEY = "primary-key";
+  public static final String PARTITION = "partition";
 
   private static final Map<String, PropertyEntry<?>> PROPERTIES_METADATA;
 
   static {
     List<PropertyEntry<?>> propertyEntries =
         ImmutableList.of(
-            stringReservedPropertyEntry(COMMENT, "The table comment", true),
```

Comment:
why remove COMMENT and CREATOR?

Reply:
the `COMMENT` property can be updated, i removed it from reserved Properties.
in Paimon, the `CREATOR` should be `OWNER`, i updated it.

Reply:
COMMENT should be updated with `updateComment` not `setProperties`

Reply:
if the key of SetProperty is `COMMENT`,  `updateComment`  works. cc @FANNG1

Reply:
I think we shouldn't allow user pass 'comment' properties as Gravitino provides `updateComment` interface

Reply:
ok